src/lib/utils/searchResults.ts

```diff
@@ -0,0 +1,17 @@
+//- Library Improts
+import { DisplayParentDomain } from 'lib/types';
+
+export const filterSearchResultsByNetwork = (
+	results: DisplayParentDomain[],
+) => {
+	if (!results) {
+		return results;
+	} else if ((process.env.REACT_APP_NETWORK ?? '') === '') {
+		return results;
+	} else {
+		const onNetworkSearchResults = results?.filter(
+			(result) => result.name.split('.')[0] === process.env.REACT_APP_NETWORK,
+		);
+		return onNetworkSearchResults;
+	}
```

Comment:
nit: could be cleaned up a little
```js
if(results && DEFAULT_NETWORK) {
    return results.filter((r) => r.name.startsWith(DEFAULT_NETWORK + '.'))
}
return results
```
_(not entirely sure if this logic is accurate or the var names are accurate)_

If you have logic statements which return the same value (in this case `results`) it's generally easier to follow if you combine them. I also like to use an early return instead of `else` when you're working with relatively simple logic and no `else if` - but that's just preference!

Reply:
Yeah that makes sense, noticed similar comments in the other open PR, will take this on board :) Thanks! Updated